projects.md

```diff
@@ -0,0 +1,10 @@
+# Projects
+
+Existing projects in Go that can be used as examples.
+
+## HelloFresh
+
+- https://github.com/hellofresh/remote-config-service
```

Comment:
I see 2 ways of doing this:
- Making this repo private if the clan agrees (which I guess won't happen)
- Moving links to internal projects to a pinned post in Slack